frontend/src/pages/modelServing/ModelServingRoutes.tsx

```diff
@@ -2,12 +2,17 @@ import * as React from 'react';
 import { Navigate, Routes, Route } from 'react-router-dom';
 import ModelServingContextProvider from './ModelServingContext';
 import ModelServingGlobal from './screens/global/ModelServingGlobal';
+import ModelServingMetrics from './screens/metrics/ModelServingMetrics';
 
 const ModelServingRoutes: React.FC = () => {
   return (
     <Routes>
       <Route path="/" element={<ModelServingContextProvider />}>
         <Route index element={<ModelServingGlobal />} />
+        <Route
+          path="/metrics/:servingruntime/:inferenceservice"
```

Comment:
What if there are serving runtimes and inference services with the same name but in different projects, could that happen?

Reply:
Yeah, that can happen -- I think we need a project name maybe?